src/include/daos/tests_lib.h

```diff
@@ -42,6 +42,14 @@
 	if (__rc != 0) \
 		fail_msg("Not successful!! Error code: " DF_RC, DP_RC(__rc)); \
 	} while (0)
+#define assert_rc_equal(rc, expected_rc) \
```

Comment:
Add a space between these definitions

Reply:
Also, minor nit.   I like to align the \'s.  It's a pain for coding but looks nicer and a little easier to inspect

Reply:
what do you mean by 's?

Reply:
It removed the \ from that